test/types/test_crd_types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package testsuites
+package testtypes
```

Comment:
These appear to be test-specific wrappers around standard k8s types and no CRD. Consider `resource_types.go` or `stdtypes.go` as a name.

Reply:
Fixed